apps/www/registry/default/example/scroll-area-horizontal-demo.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react"
+import Image from "next/image"
+
+import { ScrollArea, ScrollBar } from "@/registry/default/ui/scroll-area"
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from "@/registry/default/ui/tooltip"
+
+export interface Artwork {
+  artist: string
+  art: string
+}
+
+export const works: Artwork[] = [
+  {
+    artist: "Ornella Binni",
+    art: "https://images.unsplash.com/photo-1465869185982-5a1a7522cbcb?auto=format&fit=crop&w=300&q=80",
+  },
+  {
+    artist: "Tom Byrom",
+    art: "https://images.unsplash.com/photo-1548516173-3cabfa4607e9?auto=format&fit=crop&w=300&q=80",
+  },
+  {
+    artist: "Vladimir Malyavko",
+    art: "https://images.unsplash.com/photo-1494337480532-3725c85fd2ab?auto=format&fit=crop&w=300&q=80",
+  },
+]
+
+export default function ScrollAreaHorizontalDemo() {
+  return (
+    <ScrollArea className="w-96 whitespace-nowrap rounded-md border">
+      <div className="flex space-x-4 pb-4">
+        {works.map((artwork) => (
+          <div
+            key={artwork.artist}
+            className="h-[400px] w-[300px] overflow-hidden rounded-md"
+          >
+            <Tooltip>
```

Comment:
Let's remove the toolip and add the credit under the images as caption. This makes the code simpler.

Reply:
Updated! This is the new visual:


https://github.com/shadcn-ui/ui/assets/9381099/f6ccfdc6-f4bc-4c81-b018-1bb77fde4729